test/OpenTelemetry.Exporter.Geneva.Tests/GenevaLogExporterTests.cs

```diff
@@ -228,6 +228,99 @@ public void TableNameMappingTest(params string[] category)
             }
         }
 
+        [Fact]
+        [Trait("Platform", "Any")]
+        public void PassThruTableMappingsWhenTheRuleIsEnabled()
+        {
+            var userInitializedCategoryToTableNameMappings = new Dictionary<string, string>
+            {
+                ["Company.Store"] = "Store",
+                ["Company.Orders"] = "Orders",
+                ["*"] = "*",
+            };
+
+            var expectedCategoryToTableNameList = new List<KeyValuePair<string, string>>
+            {
+                // The category name must match "^[A-Z][a-zA-Z0-9]*$"; any character that is not allowed will be removed.
+                new KeyValuePair<string, string>("Company.Customer", "CompanyCustomer"),
+
+                new KeyValuePair<string, string>("Company-%-Customer*Region$##", "CompanyCustomerRegion"),
+
+                // If the first character in the resulting string is lower-case ALPHA,
+                // it will be converted to the corresponding upper-case.
+                new KeyValuePair<string, string>("company.Calendar", "CompanyCalendar"),
+
+                // After removing not allowed characters,
+                // if the resulting string is still an illegal Part B name, the data will get dropped on the floor.
+                new KeyValuePair<string, string>("$&-.$~!!", null),
+
+                // If the resulting string is longer than 50 characters, only the first 50 characters will be taken.
+                new KeyValuePair<string, string>("Company.Customer.rsLiheLClHJasBOvM.XI4uW7iop6ghvwBzahfs", "CompanyCustomerrsLiheLClHJasBOvMXI4uW7iop6ghvwBza"),
```

Comment:
The resulting string should be 50 characters long in this case. However, this is 49 characters long.